components/usage/cmd/run.go

```diff
@@ -44,12 +45,15 @@ func run() *cobra.Command {
 				log.WithError(err).Fatal("Failed to establish database connection.")
 			}
 
-			err = stripe.Authenticate(apiKeyFile)
-			if err != nil {
-				log.WithError(err).Fatal("Failed to initialize stripe client.")
+			log.Infof("payment integration with Stripe is set to %v", paymentEnabled)
+			if paymentEnabled {
```

Comment:
PR title says that payment is enabled when the secret is set. But here, we also set an extra flag?

Reply:
Rebased to use the existing `--api-keys-file` flag rather than introducing another one.